docs/dev/system_keyspace.md

```diff
@@ -149,6 +149,66 @@ merge the data in the legacy store with data the `truncated` table. Until the wh
 cluster agrees on the feature `TRUNCATION_TABLE` truncation will write both new and 
 legacy records. When the feature is agreed upon the legacy map is removed.
 
+## system.sstables
+
+The "ownership" table for non-local sstables
+
+Schema:
+~~~
+CREATE TABLE system.sstables (
+    location text,
+    generation bigint,
```

Comment:
It will conflict with #13994 , so let's wait for another next-promotion round to complete

Reply:
@xemul thank you Pavel, rebased and repushed.